tests/jni_api.rs

```diff
@@ -426,7 +432,7 @@ pub fn get_long_array_elements() {
     assert_eq!(res[2], 4);
 }
 
-#[test]
+//#[test]
```

Comment:
I suppose marking test as ignored (`#[ignore]`) would be better. Ideally an explaining comment should be added too.

Reply:
Sure. I didn't knew about `#[ignore]`, thanks.